docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,392 @@
+              java.naming.security.credentials: ${wls-password}
+ 
+    # Add all consumer resources-related configurations below incoming 
+    incoming:
+      #Identifier "from-wls-q" is used with @Incoming annotation
+      from-wls-q:
+        #Connector Name as specific in connector section. It is predefined
+        connector: helidon-jms
+        #JMS Destination Name in JNDI format
+        jndi.destination: queuejndi
+        #JMS Destination Type
+        type: queue
+        #JMS Connection Factory 
+        jndi.jms-factory: qcf
+ 
+    # Add all producer resources-related configurations below outgoing 
+    outgoing:
+      #Identifier "to-wls-q" is used with @Outgoing annotation
+      to-wls-q:
+        connector: helidon-jms
+        jndi.destination: queuejndi
+        type: queue
+        jndi.jms-factory: qcf
+----
+=== References [[JMS-Integration-References]]
+* https://helidon.io/docs/v3/#/mp/reactivemessaging/jms[JMS Connector]
+* https://docs-uat.us.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlshe/oracle-weblogic-server-helidon-integration.html#GUID-F7E4D3C2-6FE3-4C6E-B69B-805667B78312[Integrating Oracle WebLogic Server with Helidon]
+
+== Web Services Integration [[Web-Services-Integration]]
+
+- <<Web-Services-Integration-Overview, Overview>>
+- <<Web-Services-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<Web-Services-Integration-Usage, Usage>>
+- <<Web-Services-Integration-Example, Example>>
+- <<Web-Services-Integration-References, References>>
+
+=== Overview [[Web-Services-Integration-Overview]]
+
+Helion MP and WebLogic Server Web Services integration enables the Helidon microservice application to communicate with the WebLogic Web Service deployed in Oracle WebLogic Server.
+
+=== Maven Coordinates [[Web-Services-Integration-Maven-Coordinates]]
+
+Add the `com.oracle.webservices.wls.jaxws-wlswss-client.jar` client file downloaded from https://edelivery.oracle.com/osdc/faces/Home.jspx[Oracle Software Delivery Cloud], as part of the Maven dependencies:
+[source,]
+----
+<dependency>
+    <groupId>com.oracle.webservices.wls.jaxws-wlswss-client</groupId>
+    <artifactId>com.oracle.webservices.wls.jaxws-wlswss-client</artifactId>
+    <version>1.0</version>
+</dependency>
+----
+
+=== Usage [[Web-Services-Integration-Usage]]
+
+Use the `clientgen` WebLogic web services Ant task by using the `com.oracle.webservices.wls.jaxws-wlswss-client.jar` file, installed earlier, to generate the *jakarta* based client artifacts that client applications use to invoke the WebLogic web services. Add a call to the `clientgen` Ant task, as shown in the following example:
+[source,]
+----
+<target>
+    <path id="lib.clientgen.id">  
+        <pathelement path="${jaxws.client.jar}"/>  
+    </path> 
+ 
+    <taskdef name="clientgen" classname="weblogic.wsee.tools.anttasks.ClientGenTask" classpathref="lib.clientgen.id"/>
+    <clientgen wsdl="${remote.wsdl.location}" wsdlLocation="${remote.wsdl.location}"
```

Comment:
It's not clear what the command parameters are. It would be good to explain what's going on in general.